### PR TITLE
Add a note about info and update goals requiring at least Quarkus 2.x

### LINKED ISSUE #24520: quarkus:info/update is Quarkus 2.x series specific, documentation doesn't mention that
### Describe the bug

quarkus:info/update is Quarkus 2.x series specific, documentation doesn't mention that

a note about 1.x not being supported would make sense

### Expected behavior

documentation mention that quarkus:info/update is Quarkus 2.x series specific 

### Actual behavior

quarkus:info/update is Quarkus 2.x series specific, documentation doesn't mention that

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by aloubyansky**
Fix #24520

### CODE REVIEW

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 1]
```diff
@@ -956,6 +956,8 @@ the `-f` argument, if it was specified.
 [[project-info]]
 == Quarkus project info
 
+NOTE: This goal was introduced in Quarkus Maven plugin 2.7.0.Final and could be used in projects that are based on Quarkus version 2.0.0.Final or later.
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc:959]:
```suggestion
NOTE: This goal was introduced in Quarkus Maven plugin 2.7.0.Final and can be used in projects that are based on Quarkus version 2.0.0.Final or later.
```

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 2]
```diff
@@ -1031,6 +1033,8 @@ NOTE: `quarkus:info` will also report Quarkus extensions that aren't found in th
 [[project-update]]
 == Quarkus project update
 
+NOTE: This goal was introduced in Quarkus Maven plugin 2.7.0.Final and could be used in projects that are based on Quarkus version 2.0.0.Final or later.
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc:1036]:
```suggestion
NOTE: This goal was introduced in Quarkus Maven plugin 2.7.0.Final and can be used in projects that are based on Quarkus version 2.0.0.Final or later.
```

**Review by gsmet — COMMENTED** [review]
Small suggestion.

**Review by rsvoboda — APPROVED** [review]

**Comment by aloubyansky** [comment]
Thanks @gsmet. Squashed.

**Pull request merged**